src/main/java/org/jabref/model/entry/event/FieldChangedEvent.java

```diff
@@ -61,10 +61,12 @@ public FieldChangedEvent(FieldChange fieldChange) {
     private int computeDelta(String oldValue, String newValue) {
         if (oldValue == newValue) {
             return 0;
-        } else if (oldValue == null && newValue != null) {
+        } else if ((oldValue == null) && (newValue != null)) {
             return newValue.length();
-        } else if (newValue == null && oldValue != null) {
+        } else if ((newValue == null) && (oldValue != null)) {
             return oldValue.length();
+        } else if ((Math.abs(newValue.length() - oldValue.length()) == 0) && !oldValue.equals(newValue)) {
+            return newValue.length();
```

Comment:
How is the `delta` value used? Maybe we need to calculate the actual difference between the strings here? I.e. the delta of `aaaa` and `aabb` is 2 and not 4

Reply:
@tobiasdiez  The delta value is only used here to check if it's >1
https://github.com/JabRef/jabref/blob/a0ca87530991900db5b34df3683aa010dffe97c8/src/main/java/org/jabref/logic/util/CoarseChangeFilter.java#L49-L51

Reply:
Ok, thanks for looking this up. Then I would propose to replace `delta` by `isMajorChange`.